playbooks/roles/edxapp/tasks/service_variant_config.yml

```
@@ -1,5 +1,12 @@
 ---
 
+- name: ensure config directory exists
+  file:
+    path: "{{ EDXAPP_CFG_DIR }}"
+    state: "directory"
+    owner: "root"
+    group: "root"
```

Looking at the "copy Mongo CA certificate" task, I think we could change it to:

```suggestion
    owner: "edxapp"
    group: "www-data"
```